tests/e2e/util.go

```diff
@@ -79,6 +79,7 @@ import (
 
 	snapc "github.com/kubernetes-csi/external-snapshotter/client/v4/apis/volumesnapshot/v1"
 	snapclient "github.com/kubernetes-csi/external-snapshotter/client/v4/clientset/versioned"
+
```

Comment:
can you remove this change as its not doing anything.

Reply:
I've removed the line. JFYI, https://pkg.go.dev/golang.org/x/tools/cmd/goimports is what Intellij based IDE probably use to auto group imports by package on every save.